template/example/nuxt.config.js

```diff
@@ -8,6 +8,6 @@ module.exports = {
     resourceHints: false
   },
   modules: [
-    '@@'
+    { handler: require('@@') }
```

Comment:
This only works when running example in the jest. So i suggest using a standard path to make example working standalone too.

Reply:
Can be `require('..')`?

Reply:
@pi0 you're right!